frontend/src/views/NegotiationView.vue

```diff
@@ -21,43 +21,27 @@
 
 <script>
 export default {
-  inject: ["baseUrl"],
   data() {
     return {
+      backendURL: import.meta.env.VITE_BACKEND_BASE_URL,
       catalog: {},
     };
   },
   methods: {
-    getCatalog() {
-      let vm = this;
-      fetch(vm.baseUrl + "/edc/negotiations")
+    getNegotiations() {
+      fetch(this.backendURL + "/edc/negotiations", {
```

Comment:
With the default settings, this.backendURL has a trailing slash. Therefore a leading slash in front of "/edc/negotiations" is not working correctly.

Reply:
fixed